plugins/core/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -38,6 +38,7 @@ action.aws.toolkit.toolwindow.newConnection.text=Add Another Connection...
 action.dynamic.open.text=Open Resource...
 action.q.openchat.text=Open Chat Panel
 amazonqChat.project_context.index_in_progress=By the way, I'm still indexing this project for full context from your workspace. I may have a better response in a few minutes when it's complete if you'd like to try again then.
+amazonqChat.stopChatResponse=You stopped your current work, please provide additional examples or ask another question.
```

Comment:
should use amazonq bundle instead @leigaol

Reply:
`AmazonQBundle.properties`

Reply:
There are lots of new messages checked in the core bundle... I was following previous commits. 

Can do as a fast follow. This needs to go in for today's bug bash. I have 2nd PR depending on this!